BraveWallet/Crypto/NetworkPicker.swift

```diff
@@ -111,43 +81,35 @@ struct NetworkPicker: View {
       )
       .clipShape(Capsule())
       .contentShape(Capsule())
-      .animation(nil, value: networkStore.selectedChain)
-    }
-    .accessibilityLabel(Strings.Wallet.selectedNetworkAccessibilityLabel)
-    .accessibilityValue(networkStore.selectedChain.shortChainName)
-    .sheet(isPresented: $isPresentingAddNetwork) {
-      NavigationView {
-        CustomNetworkDetailsView(networkStore: networkStore, model: .init())
-      }
-    }
-    .alert(
-      isPresented: $isShowingNextNetworkAlert
-    ) {
-      Alert(
-        title: Text(String.localizedStringWithFormat(Strings.Wallet.createAccountAlertTitle, nextNetwork?.shortChainName ?? "")),
-        message: Text(Strings.Wallet.createAccountAlertMessage),
-        primaryButton: .default(Text(Strings.yes), action: {
-          // show create account for `nextNetwork.coin`
-          isPresentingAddAccount = true
-        }),
-        secondaryButton: .cancel(Text(Strings.no), action: {
-          // not creating account, don't switch to nextNetwork
-          self.nextNetwork = nil
-        })
-      )
-    }
-    .sheet(
-      isPresented: $isPresentingAddAccount
-    ) {
-      NavigationView {
-        AddAccountView(keyringStore: keyringStore, preSelectedCoin: nextNetwork?.coin)
-      }
-      .onDisappear {
-        // User either created an account, or cancelled. If account created,
-        // `NetworkStore`s `keyringCreated` observation will switch to the new account
-        self.nextNetwork = nil
-      }
     }
+    .animation(nil, value: networkStore.selectedChain)
+    .background(
+      Color.clear
+        .sheet(isPresented: $isPresentingAddNetwork) {
+          NavigationView {
+            CustomNetworkDetailsView(networkStore: networkStore, model: .init())
+          }
+        }
+    )
+    .background(
+      Color.clear
+        .sheet(
+          isPresented: Binding(
+            get: { self.networkSelectionStore != nil },
+            set: { if !$0 { self.networkSelectionStore = nil } }
+          )
+        ) {
+          if let networkSelectionStore = networkSelectionStore {
+            NavigationView {
```

Comment:
Does this need `stack` style since you have possible nav links inside?

Reply:
Fixed in [c29f314](https://github.com/brave/brave-ios/pull/5706/commits/c29f314f45ae50559626c245a8c40fe2dbdfd34e)